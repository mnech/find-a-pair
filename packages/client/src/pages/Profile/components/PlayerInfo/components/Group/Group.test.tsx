import React from 'react';
import { render } from '@testing-library/react';
import { Group } from './index';

describe('Group profile component', () => {
  describe('should be rendered correctly Group component', () => {
    test('default', () => {
      const { asFragment } = render(<Group name="Name" value="value" />);
      expect(asFragment()).toMatchInlineSnapshot(`
        <DocumentFragment>
          <div
            class="group"
          >
            <span>
              Name
            </span>
            <span>
              value
            </span>
          </div>
        </DocumentFragment>
      `);
    });
  });
});
