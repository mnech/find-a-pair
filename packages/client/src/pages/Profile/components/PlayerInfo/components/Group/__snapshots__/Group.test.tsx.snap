// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Group profile component should be rendered correctly Group component default 1`] = `
<DocumentFragment>
  <div
    class="group"
  >
    <span>
      Name
    </span>
    <span>
      value
    </span>
  </div>
</DocumentFragment>
`;
